content/en/synthetics/browser_tests/actions.md

```diff
@@ -304,69 +352,115 @@ You can run HTTP requests as part of your browser tests.
+   * Extract the value using the **full response body**.
 
-{{< img src="synthetics/browser_tests/extracted_variable.png" alt="Extracted variable from response" style="width:80%;">}}
+{{< img src="synthetics/browser_tests/extracted_variable.png" alt="Extracted variable from response" style="width:80%;" >}}
 
-To see all available variables, type `{{` in the input field and choose your variable of interest.
-
-Once created, this variable can be used in the [following steps](#use-variables) of your browser test.
+You can extract up to ten variables per test step. Once created, this variable can be used in the [following steps](#use-variables) of your browser test.
 
 ## Use variables
 
 To see all available variables on manually added steps, type `{{` in the input field.
 
-If you want to use a variable on automatically recorded steps, click the **Inject this variable** icon:
+To use a variable on automatically recorded steps, click the **Inject this variable** icon to input the variable value in the recorder page while recording. 
 
-{{< img src="synthetics/browser_tests/variable_input.mp4" alt="Variable Input" video="true" width="100%" >}}
+{{< img src="synthetics/browser_tests/variable_input.mp4" alt="Click on the test step to inject the value in your recorder page" video="true" width="100%" >}}
 
-At recording, this translates into the actual value of the variable being injected on your website's input (consequently allowing you to move on with the rest of your steps) and creates an associated `Type text` step featuring `{{ <YOUR_VARIABLE_NAME> }}`. At test execution, `{{ <YOUR_VARIABLE_NAME> }}` is systematically replaced by your variable's associated value.
+If a variable is assigned different values along your browser test steps (for example, across subtests), the variable systematically uses the value that was first assigned to it.
 
-Some variables are only computed at runtime (for example, a variable from an HTTP request or from a JavaScript step). To record a step using one of these variables, record a step with the actual variable value, then replace the actual value with `{{ <YOUR_VARIABLE_NAME> }}` on your step definition before saving your test. 
+Some variables only compute at runtime, such as a variable from an HTTP request or a JavaScript step. For example, assume you have a `Type text` step featuring `{{ <YOUR_VARIABLE_NAME> }}`. At test execution, `{{ <YOUR_VARIABLE_NAME> }}` is systematically replaced by your variable's associated value.
 
-If a variable is assigned different values along your browser test steps (for example, across subtests), the variable systematically uses the value that was first assigned to it.
+To record a step using one of these variables, record a step with the actual variable value, and replace the actual value with `{{ <YOUR_VARIABLE_NAME> }}` in the step's definition before saving your test. 
```

Comment:
can we have the last 2 paragraphes as one?
maybe we can keep a return to line but we shouldn't have 2 paragraphes